docs/usage-guide.md

```diff
@@ -66,15 +66,20 @@ Chroma uses collection names in the url, so there are a few restrictions on nami
 - The name must not contain two consecutive dots.
 - The name must not be a valid IP address.
 
-Chroma collections are created with a name and an optional embedding function.
+Chroma collections are created with a name and an optional embedding function. If you supply an embedding function, you must supply it every time you get the collection.
 
 ```python
-collection = client.create_collection(name="my_collection", embedding_function=emb_fn, metric="dot", dimension=768)
+collection = client.create_collection(name="my_collection", embedding_function=emb_fn)
+collection = client.get_collection(name="my_collection", embedding_function=emb_fn)
 ```
 
+:::caution
+If you later wish to `get_collection`, you MUST do so with the embedding function supplied.
```

Comment:
maybe 

"with the embedding function you supplied on creating the collection"

Reply:
Ah woops. Thanks!